cmd/linkflags/main.go

```diff
@@ -54,7 +57,11 @@ func run() error {
 	log.SetFlags(0)
 	flag.Parse()
 	if *pkg == "" {
-		return fmt.Errorf("-pkg required")
+		dir, err := filepath.Abs(filepath.Dir(os.Args[0]))
```

Comment:
I think it will resolve to symlink location, but users can always set it explicitly to make sure everything works